odoo/http.py

```diff
@@ -1329,7 +1329,7 @@ def load_addons(self):
                     path_static = opj(addons_path, module, 'static')
                     if manifest_path and os.path.isdir(path_static):
                         manifest_data = open(manifest_path, 'rb').read()
-                        manifest = ast.literal_eval(pycompat.to_native(manifest_data))
+                        manifest = ast.literal_eval(manifest_data)
```

Comment:
Either the file needs to be opened in text mode (with an encoding provided) or the contents need to be decoded after read, I don't think literal_eval can take a bytes input